### PR TITLE
CCS-4453 - Improvement on existing code

### PR DESCRIPTION
**Pull request by rednitish**
Added support for providing environment as argument to help testers/devs have it working on other environments.

Added print usages instructions for better understanding of script.

### CODE REVIEW

File: scripts/populateDrupalCache.py [hunk 1]
```diff
@@ -2,16 +2,33 @@
...
+            'https://pantheon.corp.stage.redhat.com/api/sitemap/assembly.sitemap.xml',
+            'https://pantheon.corp.stage.redhat.com/api/sitemap/module.sitemap.xml'
+        ],
+        "dev" : [
+            'https://pantheon.corp.dev.redhat.com/api/sitemap/assembly.sitemap.xml',
+            'https://pantheon.corp.dev.redhat.com/api/sitemap/module.sitemap.xml'
+        ]})
 
-def gather_urls():
+headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36'}
```
Comment (aprajshekhar) [inline scripts/populateDrupalCache.py:25]: Query: Do we need headers?

**Comment by aprajshekhar** [comment]
LGTM. Just one query.

File: scripts/populateDrupalCache.py (hunk 1 above)
Comment (rednitish) [reply scripts/populateDrupalCache.py:25]: @aprajshekhar It is not helping us, i kept to to get rid off forbidden 403. But on preprod, we are still getting it. So we can remove it.

**Review by aprajshekhar — APPROVED** [review]
LGTM

**Pull request merged**